docs-source/content/security/openshift.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "OpenShift"
+date: 2019-10-04T08:08:08-05:00
+weight: 7
+description: "OpenShift information for the WebLogic operator"
+---
+
+#### OpenShift `anyuid` security context
+
+The Docker images that Oracle publishes default the container user
+as `oracle` which is UID `1000` and GID `1000`. When running the
+Oracle images or layered images that retain the default user as
```

Comment:
Is there a general discussion of the docker image user, job/pod security context, and pv/c security context settings somewhere?  It'd be nice to have a place that ties them all together.

Reply:
Enter an issue for the docs?

Reply:
Please enter a documentation issue for the suggested as this is not specifically about providing more details on the `anyuid` security context for OpenShift.